assets/src/features/home-page/components/HomePageLayout.tsx

```diff
@@ -7,15 +7,15 @@ const HomePageLayout: React.FC<PropsWithChildren> = ({ children }) => {
   return (
     <div
       className={clsx(
-        "home-page h-screen w-full bg-brand-sea-blue-200 text-brand-dark-blue-500",
+        "home-page h-[100dvh] w-full bg-brand-sea-blue-200 text-brand-dark-blue-500",
         "font-rocGrotesk",
         "flex flex-col items-center gap-y-4 p-4",
-        "relative"
+        "relative overflow-y-auto"
```

Comment:
It still doesn't feel right. I inspected the design more closely and the distance from the top of the page to h2 is the same as from the content (preview and form) to bottom

![image](https://user-images.githubusercontent.com/2119556/219114107-f0c366d6-15e8-4a13-acee-237aa862ff05.png)

(it was more visible on smaller screens, the screenshot is zoomed)

Making the logo `position: absolute` seems to improve that matter although that might not be the best solution as I'm not sure how will it affect mobiles for example